table/src/main/java/com/bazaarvoice/emodb/table/db/test/InMemoryTableDAO.java

```diff
@@ -42,6 +45,16 @@
         return _uuidToTable.values().iterator();
     }
 
+    @Override
+    public Iterator<ChangedTable> listMetadataChangedTables(DateTime fromInclusive, DateTime toExclusive, LimitCounter limit) {
+        return null;
```

Comment:
Maybe throw an `UnsupportedOperationException()` instead.  It'd be easier to catch errors in other tests which may call this method than to have a null pointer exception.